src/ltest-listener.lfe

```diff
@@ -76,15 +76,21 @@
     state))
 
 (defun handle_cancel
-  ((_ data state)
-    state))
+  ((_group_or_test data state)
+    (ltest-formatter:display-test-cancel (proplists:get_value 'reason data))
+    ;;This counts groups and tests together
+    (set-state-err state (+ 1 (state-err state)))))
 
 (defun terminate
   ((`#(ok ,data) state)
     ;(io:format "Terminating ...~n")
```

Comment:
you mean the `;(io:format...)` comment? It was there before my patch, so I decided not to fix the comments that were already there - if you mean the `;;This counts groups....` line then I misunderstood you - let me know if this was the case.